.github/workflows/rust.yml

```
@@ -21,7 +21,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - run: rustup update
-    - run: find . -type f -name '*.rs' -print0 | xargs -I {} -0 rustfmt --check "{}"
+    - run: find . -type f -name '*.rs' -print0 | xargs -I {} -0 rustfmt --edition 2021 --check "{}"
```

We can use this now:
```suggestion
    - run: cargo fmt --all --check
```